src/InlineInput/InlineInput.tsx

```diff
@@ -142,12 +144,12 @@ const InlineInput = React.forwardRef<HTMLInputElement, Props>(
             <input
                 ref={ref}
                 type="text"
-                className={classNames(
+                className={`${classNames(
                     'inline-input',
                     isValid(internalValue) || 'invalid',
                     disabled && 'disabled',
                     className
-                )}
+                )} ${textAlignLeft && 'tw-pl-2 tw-text-left'}`}
```

Comment:
If `textAlignLeft` is `false`, the string `false` will be added to the class names.

Better use the `classNames` function, which is already there anyhow, like this:
```js
className={classNames(
  'inline-input',
  isValid(internalValue) || 'invalid',
  disabled && 'disabled',
  textAlignLeft && 'tw-pl-2 tw-text-left'
  className
)}
```